ansible/roles/go-ipfs-deploy-bin/templates/ipfs.logrotate.j2

```diff
@@ -0,0 +1,7 @@
+{{ go_ipfs_debug_log }} {
+  daily
+  missingok
+  rotate 2
```

Comment:
might be worth keeping more than 2 days of logs. Should ship them out anyways.

Reply:
sure but out of scope right for this PR. please create separate task